cloudimg/aws.py

```diff
@@ -240,6 +240,32 @@ def get_image_by_filters(self, filters):
 
         return self.ec2.Image(images[0]['ImageId'])
 
+    def get_image_from_ami_catalog(self, image_id):
+        """
+        Finds Image in AMI Catalog with given filters.
+        The search includes AMIs owned by self, AWS marketplace AMIs,
+        and Community AMIs.
+
+        Args:
+            filters(list): List of filters (dict) to apply.
```

Comment:
This doesn't match the method's actual signature, there is no list of filters (dict).

Reply:
fixed it.